src/workspace.coffee

```diff
@@ -371,9 +371,11 @@ class Workspace extends Model
   Section: Opening
   ###
 
-  # Essential: Open a given a URI in Atom asynchronously.
-  #
-  # * `uri` A {String} containing a URI.
+  # Essential: Open a given URI in Atom asynchronously. 
+  # If no URI is given, or URI does not resolve to an existing file, 
```

Comment:
*First: I have been looking at atom for three spare nights now. I have very little knowledge of how it works.*

What I found, was that if we have `atom.workspace.open("file_not_on_disk.txt")`, we get a new editor/document where the filename is initalized to the URI, so if you hit `save` you wan't be prompted for a filename - it will just save the file in the root of the current project. 
If you do not have a project open, you will still get a new document, but the document will not get the filename specified. It will get 'untitled'. 

So the proposed text `.., or URI does not resolve to an existingn file, ...` was supposed to capture some of this behavior. Not a very clear formulation I guess.. :flushed: ..but that was it's intent.

I think maybe your proposed change will document yet another behavior that I was unaware of. Maybe it should be added as a separate bullet point? (The documentation guidelines may have an opinion on how behavior should be listed?)